scripts/deploy-to-s3.sh

```diff
@@ -2,7 +2,7 @@
 
 # ============================  Script Variables   ============================
 DEPLOY_BUCKET="bart.gariany.com"
-PUBLIC_FOLDER="../public"
+PUBLIC_FOLDER="./public"
```

Comment:
With these changes in place CircleCI works but if you try to cd into the script folder and run the script it will not be able to locate `./src` or `./public` (as they are one stop above the folder).

Reply:
Ok so what should we do about this ?